[BUG] Debugging in Intellij IDEA 2026.1 fails

## 📚 Before Submitting

**Please check our [documentation](https://genie.devoxx.com/docs/getting-started/introduction) first!**

You may also find helpful information in our [Features Overview](https://genie.devoxx.com/docs/features/overview) or search [existing issues](https://github.com/devoxx/DevoxxGenieIDEAPlugin/issues).

## Bug Description

**Describe the bug:**
Debugging the plugin in IntelliJ IDEA 2026.1 fails due to an outdated dependency configuration in the Gradle build script. The `intellij-platform-gradle-plugin` no longer supports the `ideaIC` artifact identifier for versions 2025.3 and later.

## Steps to Reproduce

1. Configure the project to run with IntelliJ IDEA 2026.1.
2. Execute the `runIde` task (e.g., `./gradlew runIde -PideVersion=2026.1`).
3. Observe the build failure.

## Expected Behavior

The `runIde` task should successfully download the appropriate IntelliJ Platform dependency and launch the IDE for debugging.

## Actual Behavior

The build fails with:
`[org.jetbrains.intellij.platform] IntelliJ IDEA Community (IC) is no longer published since 2025.3 (253), use: intellijIdea("2026.1")`
`Could not find idea:ideaIC:2026.1.`

## Screenshots/Logs

```
[org.jetbrains.intellij.platform] IntelliJ IDEA Community (IC) is no longer published since 2025.3 (253), use: intellijIdea("2026.1")
[...]
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':runIde'.
> Failed to query the value of task ':runIde' property 'runtimeDirectory'.
   > Could not resolve all files for configuration ':intellijPlatformDependency'.
      > Could not find idea:ideaIC:2026.1.
```

## Environment

- **IntelliJ IDEA Version**: 2026.1
- **Plugin Version**: 1.6.0
- **Operating System**: Ubuntu 24.04.4 LTS
- **LLM Provider**: N/A
- **Model**: N/A

## Additional Context

This is a breaking change in the IntelliJ Platform Gradle Plugin. The build script needs to be updated to use the `intellijIdea()` helper method instead of explicitly referencing `ideaIC` when the target version is 2025.3 or higher.

## Configuration (if relevant)

- **Are you using MCP servers?** No
- **Is RAG enabled?** No
- **Any custom prompts or skills configured?** No

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -7,7 +7,7 @@ plugins {
     kotlin("jvm") version "2.3.21"
     kotlin("plugin.lombok") version "2.3.21"
     kotlin("plugin.compose") version "2.3.21"
-    id("org.jetbrains.intellij.platform") version "2.13.1"
+    id("org.jetbrains.intellij.platform") version "2.15.0"
     jacoco
 }
 
@@ -35,12 +35,7 @@ val binaryIncompatibleRuntimeJarPatterns = listOf(
     "kotlinx-coroutines-core-jvm-*.jar"
 )
 val packagedPluginDirName = "DevoxxGenie"
-val pluginVerifierCommunityIdeVersions = listOf(
-    "2025.1.7",   // 251 line
-    "2025.2.6.1"  // 252 line
-)
 val pluginVerifierUnifiedIdeVersions = listOf(
-    "2025.3.3",          // 253 line
     "2026.1",             // 261 line
     "2026.2-EAP-SNAPSHOT" // 262 line
 )
@@ -378,8 +373,8 @@ dependencies {
 intellijPlatform {
     pluginConfiguration {
         ideaVersion {
-            sinceBuild = "251"
-            untilBuild = "262.*"
+            sinceBuild = "261"
+            untilBuild = "261.*"
         }
     }
 
@@ -395,9 +390,6 @@ intellijPlatform {
 
     pluginVerification {
         ides {
-            pluginVerifierCommunityIdeVersions.forEach { ideVersion ->
-                create("IC", ideVersion) {}
-            }
             pluginVerifierUnifiedIdeVersions.forEach { ideVersion ->
                 create("IU", ideVersion) {}
             }
@@ -425,11 +417,7 @@ afterEvaluate {
 
 tasks {
     // Run plugin on different IntelliJ versions for testing
-    // Usage: ./gradlew runIde -PideVersion=2024.3.5
-    //        ./gradlew runIde -PideVersion=2025.1.1
-    //        ./gradlew runIde -PideVersion=2025.2.2
-    //        ./gradlew runIde -PideVersion=2025.3.3
-    //        ./gradlew runIde -PideVersion=2026.1
+    // Usage: ./gradlew runIde -PideVersion=2026.1
     //        ./gradlew runIde -PideVersion=2026.2-EAP-SNAPSHOT
 
     withType<Jar> {
@@ -484,9 +472,8 @@ tasks {
     }
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+kotlin {
+    jvmToolchain(25)
 }
 
 kotlinLombok {
@@ -495,6 +482,6 @@ kotlinLombok {
 
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
     compilerOptions {
-        jvmTarget.set(org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_17)
+        jvmTarget.set(org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_25)
     }
 }

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -24,7 +24,7 @@ repositories {
 }
 
 jacoco {
-    toolVersion = "0.8.12"
+    toolVersion = "0.8.14"
 }
 
 val binaryIncompatibleRuntimeJarPatterns = listOf(

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -4,9 +4,9 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     java
-    kotlin("jvm") version "2.1.10"
-    kotlin("plugin.lombok") version "2.1.10"
-    kotlin("plugin.compose") version "2.1.10"
+    kotlin("jvm") version "2.3.21"
+    kotlin("plugin.lombok") version "2.3.21"
+    kotlin("plugin.compose") version "2.3.21"
     id("org.jetbrains.intellij.platform") version "2.13.1"
     jacoco
 }
@@ -18,6 +18,7 @@ repositories {
     mavenCentral()
     intellijPlatform {
         defaultRepositories()
+        jetbrainsRuntime()
     }
     google()
 }
@@ -40,6 +41,7 @@ val pluginVerifierCommunityIdeVersions = listOf(
 )
 val pluginVerifierUnifiedIdeVersions = listOf(
     "2025.3.3",          // 253 line
+    "2026.1",             // 261 line
     "2026.2-EAP-SNAPSHOT" // 262 line
 )
 
@@ -222,8 +224,8 @@ tasks.named("processResources") {
 
 dependencies {
     intellijPlatform {
-        // Allow overriding IDE version via property: ./gradlew runIde -PideVersion=2025.1.7
-        create("IC", providers.gradleProperty("ideVersion").orElse("2025.1.7")) {}
+        // Allow overriding IDE version via property: ./gradlew runIde -PideVersion=2026.1.1
+        intellijIdea(providers.gradleProperty("ideVersion").orElse("2026.1"))
         bundledPlugin("com.intellij.java")
         bundledPlugin("org.intellij.plugins.markdown")  // Required by markdown renderer
         composeUI()
@@ -427,6 +429,7 @@ tasks {
     //        ./gradlew runIde -PideVersion=2025.1.1
     //        ./gradlew runIde -PideVersion=2025.2.2
     //        ./gradlew runIde -PideVersion=2025.3.3
+    //        ./gradlew runIde -PideVersion=2026.1
     //        ./gradlew runIde -PideVersion=2026.2-EAP-SNAPSHOT
 
     withType<Jar> {

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -39,7 +39,8 @@ val pluginVerifierCommunityIdeVersions = listOf(
     "2025.2.6.1"  // 252 line
 )
 val pluginVerifierUnifiedIdeVersions = listOf(
-    "2025.3.3"    // 253 line
+    "2025.3.3",          // 253 line
+    "2026.2-EAP-SNAPSHOT" // 262 line
 )
 
 fun Project.stripBinaryIncompatibleRuntimeJars(sandboxPluginPath: String) {
@@ -376,7 +377,7 @@ intellijPlatform {
     pluginConfiguration {
         ideaVersion {
             sinceBuild = "251"
-            untilBuild = "261.*"
+            untilBuild = "262.*"
         }
     }
 
@@ -426,6 +427,7 @@ tasks {
     //        ./gradlew runIde -PideVersion=2025.1.1
     //        ./gradlew runIde -PideVersion=2025.2.2
     //        ./gradlew runIde -PideVersion=2025.3.3
+    //        ./gradlew runIde -PideVersion=2026.2-EAP-SNAPSHOT
 
     withType<Jar> {
         duplicatesStrategy = DuplicatesStrategy.EXCLUDE

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -11,7 +11,7 @@
     <!-- A displayed Vendor name or Organization ID displayed on the Plugins Page. -->
     <vendor email="info@devoxx.com" url="https://devoxx.com">Devoxx</vendor>
 
-    <idea-version since-build="243" until-build="262.*"/>
+    <idea-version since-build="262" until-build="262.*"/>
 
     <!-- Description of the plugin displayed on the Plugin Page and IDE Plugin Manager.
          Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -11,7 +11,7 @@
     <!-- A displayed Vendor name or Organization ID displayed on the Plugins Page. -->
     <vendor email="info@devoxx.com" url="https://devoxx.com">Devoxx</vendor>
 
-    <idea-version since-build="243" until-build="261.*"/>
+    <idea-version since-build="243" until-build="262.*"/>
 
     <!-- Description of the plugin displayed on the Plugin Page and IDE Plugin Manager.
          Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.

